frameworks/helloworld/src/main/dist/multiport.yml

```diff
@@ -0,0 +1,65 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+  user: {{FRAMEWORK_USER}}
+pods:
+  multiport:
+    count: {{HELLO_COUNT}}
+    resource-sets:
+      multi-port-resources:
+        ports:
+          port_one:
+            port: {{HELLO_PORT_ONE}}
```

Comment:
See my comment on simply hard-coding the ports here, leaving them to be automatically assigned, or both.